src/current/_includes/common/upgrade/overview.md

```diff
@@ -12,7 +14,7 @@
     A major version of CockroachDB has two phases of patch releases: a series of **testing releases** (beta, alpha, and RC releases) followed by a series of **production releases** (vX.Y.0, vX.Y.1, and so on). A major version’s first production release (the .0 release) is also known as its GA release.
 
 {% if page.path contains "cockroachcloud" %}
-    For CockroachDB {{ site.data.products.advanced }}, {{ site.data.products.standard }}, and {{ site.data.products.basic }} clusters, all production patch releases for a major version are automatically applied, until the cluster is upgraded to a new major version.
+    For all CockroachDB {{ site.data.products.cloud }} clusters, production patch releases for a major version are automatically applied until the cluster is upgraded to a new major version.
```

Comment:
Add context that you have the option to defer patches for 30/60/90 days if you have maintenance windows set on Advanced cluster.